### PR TITLE
Bulk spelling & indentation fixes

### PR DESCRIPTION
**Pull request by hendursaga**
Lots of small spelling fixes, removing Britishisms, and re-indenting those files while connected to a running instance.

### CODE REVIEW

File: source/buffer.lisp [hunk 1]
```diff
@@ -636,8 +636,8 @@ store them somewhere and `ffi-buffer-delete' them once done."))
 
 (define-class status-buffer (input-buffer)
   ((window nil
-    :type (maybe window)
-    :documentation "The `window' to which the status buffer is attached.")
+           :type (maybe window)
+           :documentation "The `window' to which the status buffer is attached.")
```
Comment (Ambrevar) [inline source/buffer.lisp:640]: in this case we should insert a newline before the initform `nil` instead.

**Review by Ambrevar — COMMENTED** [review]
Excellent!  Just one nit and we can merge!

File: source/buffer.lisp (hunk 1 above)
Comment (hendursaga) [reply source/buffer.lisp]: Fixed! You can (optionally) squash and then merge!

**Comment by Ambrevar** [comment]
Merged with ccfdbfafb6d93fe550cfec88fc603ddc1ad73c29.  Thanks a lot for this painful fix! :)

**Pull request closed without merging**